packages/kuma-gui/src/app/application/components/route-view/RouteView.vue

```diff
@@ -191,17 +206,24 @@ watch(() => {
     ...route.params,
   }
 
+  const propsParams = Object.entries(props.params).reduce((acc, [key, value]) => {
+    let param = params[key]
+    if(value === Number || typeof value === 'number') {
```

Comment:
Immediate thought here is that if the rest of the code is made to work with `Number` and `String` how come we aren't dealing with `String` here?